library/Ivoz/Provider/Domain/Service/BillableCall/UpdateByTpCdr.php

```diff
@@ -51,7 +51,7 @@ public function handle(DomainEventInterface $domainEvent)
         if ($isNotOutbound) {
             $msg = sprintf(
                 'Skipping %s call #%d',
-                $billableCall->getDirection(),
+                $billableCall->getDirection() ?? '',
```

Comment:
Just a comment not meant to be handled in this pull request.

Billable Call Direction should not be nullable. Inbound or Outbound. Other values should not be allowed.